cmd/plugin.go

```diff
@@ -409,7 +409,13 @@ func runPluginUpdateCmd(cmd *cobra.Command, args []string) {
 			ref := ociinstaller.NewSteampipeImageRef(p)
 			isExists, _ := plugin.Exists(p)
 			if isExists {
-				runUpdatesFor = append(runUpdatesFor, versionData.Plugins[ref.DisplayImageRef()])
+				if strings.HasPrefix(ref.DisplayImageRef(), "hub.steampipe.io/") {
```

Comment:
can we put a const in for `hub.steampipe.io/` in the consts package

Reply:
Done!